src/scanners/javascript.js

```diff
@@ -112,6 +112,7 @@ export default class JavaScriptScanner {
             var overwrites = ESLINT_OVERWRITE_MESSAGE[message.ruleId];
             var shortDescription = overwrites.message || message.message;
             var description = overwrites.description || message.description;
+            code = overwrites.code || message.message;
```

Comment:
`code` is already set to `message.message` so I think this could reuse `code` instead, or only set it in the case that it will change.

```js
code = overwrites.code || code;
// or
if (overwrites.code) {
  code = overwrites.code;
}